.github/workflows/check_file_size.yml

```diff
@@ -0,0 +1,18 @@
+name: Check file size
+on: pull_request
```

Comment:
Should be also do this on every push?

Reply:
Initially I added it to run it on `push`. We can change it to that if on push is better option.